go/host/rpc/clientapi/client_api_scan.go

```diff
@@ -0,0 +1,42 @@
+package clientapi
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/obscuronet/go-obscuro/go/common/host"
+)
+
+// ScanAPI implements metric specific RPC endpoints
+type ScanAPI struct {
+	host   host.Host
+	logger log.Logger
+}
+
+func NewScanAPI(host host.Host, logger log.Logger) *ScanAPI {
+	return &ScanAPI{
+		host:   host,
+		logger: logger,
+	}
+}
+
+//
+//func (s *ScanAPI) GetL1RollupHeaderByHash() (*common.RollupHeader, error) {
```

Comment:
Did you mean to leave these commented out bits?

Reply:
Removed them!